### PR TITLE
Removed workshoppers that have nothing to do with Node

### PR DESCRIPTION
**Pull request by martinheidegger**
In https://github.com/nodeschool/organizers/issues/378#issuecomment-248086701 I mentioned that there are some workshoppers here that have **nothing** to do with Node. Specifically the  [web-audio-school](https://github.com/mmckegg/web-audio-school), [intro-to-webgl](https://github.com/alexmackey/IntroToWebGLWithThreeJS), [shader-school](https://github.com/stackgl/shader-school) and [webgl-workshop](https://github.com/stackgl/webgl-workshop).

This PR removes those workshoppers and their translation in the language files. It also removes the paragraph in about that talks about this variety. _(Note: it doesn't remove the section of thinking out-of-the-box in build because I am havn't found an easy way to do it, and because I think its minor)_

### CODE REVIEW

File: about.html [hunk 1]
```diff
@@ -42,16 +42,7 @@ <h3 data-i18n="about-header-history-title">Interactive tutorials</h3>
...
-
-      <div class="container">
-        <div class="third">
-          <h3 data-i18n="about-header-philosophy-title">A wide range of topics</h3>
-        </div>
-        <div class="two-thirds">
-          <p data-i18n="about-philosophy">All of the NodeSchool workshops are written using Node, but the subject of a workshop doesn&apos;t necessarily have to be about Node. For example the <a href="https://www.npmjs.org/package/shader-school">shader-school</a> workshop teaches computer graphics concepts and doesn&apos;t involve writing JavaScript at all.</p>
-          <p data-i18n="about-philosophy2">The NodeSchool community has developed some really great tools for authoring new workshops. If you have an idea for a workshop we encourage you to check out our <a href="building-workshops.html">Building Workshops</a> page.</p>
-        </div>
+        <p data-i18n="about-hisotry5">The NodeSchool community has developed some really great tools for authoring new workshops. If you have an idea for a workshop we encourage you to check out our <a href="building-workshops.html">Building Workshops</a> page.</p>
```
Comment (riyadhalnur) [inline about.html]: Typo `about-hisotry5` -> `about-history5`

**Comment by riyadhalnur** [comment]
@martinheidegger Are we going to add a separate area for workshoppers like these, i.e the ones that are being removed?

**Comment by SomeoneWeird** [comment]
I absolutely don't think we should be removing these. Instead maybe we can
create an area for "off topic"(?) workshops

**Comment by martinheidegger** [comment]
@SomeoneWeird to be clear: I am conflicted about this removal as (mentioned in the issue https://github.com/nodeschool/organizers/issues/378#issuecomment-248086701). In part the discussion - if this is merged - needs to be held in the other issue.

@riyadhalnur I updated the PR without the typo

**Comment by martinheidegger** [comment]
Seems like this is not desired.

**Pull request closed without merging**